nodes/arduino_handler.py

```diff
@@ -273,38 +275,43 @@ def process_message(line):
                 error_message)
             rospy.logwarn(message)
             return message
-        # status: OK
+        # else status: OK
 
         # Zip values with the corresponding environmental variable
         variable_values = values[1:]
         pairs = tuple((headers[0], headers[1](value))
             for headers, value in zip(sensor_csv_headers[1:], variable_values))
         return pairs
     except ValueError:
-        message = "Type conversion error, skipping."
+        message = "arduino_handler: Type conversion error, skipping."
         rospy.logwarn(message)
         return message
     except IndexError:
-        message = "Short read, received part of a message: {}".format(buf.decode())
+        message = "arduino_handler: Partial message: >{}<".format(line)
         rospy.logwarn(message)
         serial_connection.close()
         serial_connection.open()
         return message
     # Occasionally, we get rotten bytes which couldn't decode
     except UnicodeDecodeError:
-        message = "Received weird bits, ignoring: {}".format(buf)
+        message = "arduino_handler: Ignoring weird bits: >{}<".format(line)
         rospy.logwarn(message)
         serial_connection.close()
         serial_connection.open()
         return message
 
+
+#------------------------------------------------------------------------------
```

Comment:
But, I like those 'artifacts'!  It makes it easier for me to read the code since it doesn't have {} blocks like a real language...     I will remove them if they offend you.